src/pages/HomePage.js

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
+import FeaturedProjects from '../components/FeaturedProjects';
+import collabImage from '../assets/images/collab-image.svg';
```

Comment:
Check out about using absolute imports paths. few reference 'https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8'. set it such that we can use the absolute path as `src/path_of_component`.